src/color/base.json

```diff
@@ -4,70 +4,77 @@
       "white": {
         "value": "ffffff",
         "comment": "{comments.color.base.value.comment}",
-        "legacy": true
+        "legacy": true,
+        "public": true
       },
       "white-opacity-40": {
         "value": "ffffff66",
-        "comment": "white @ 40%"
+        "comment": "white @ 40%",
+        "public": true,
+        "opacity": true
       },
       "black": {
         "value": "000000",
         "comment": "{comments.color.base.value.comment}",
-        "legacy": true
+        "legacy": true,
+        "public": true
       },
       "black-opacity-15": {
         "value": "00000026",
-        "comment": "black @ 15%"
-      },
-      "cloud": {
-        "value": "f7f7f7",
-        "comment": "{comments.color.base.value.comment}",
-        "legacy": true
+        "comment": "black @ 15%",
+        "public": true,
+        "opacity": true
       },
-      "crimson": {
+      "gray": {
         "100": {
-          "value": "ff999b"
+          "value": "f7f7f7",
+          "public": true
         },
-        "500": {
-          "value": "df0b37"
-        }
-      },
-      "disabled": {
-        "value": "dbdbdb"
-      },
-      "emerald": {
-        "100": {
-          "value": "69cf96"
+        "200": {
+          "value": "dbdbdb",
+          "public": true
         },
         "500": {
-          "value": "00805d"
+          "value": "222222",
+          "public": true
         }
       },
       "manatee": {
         "value": "9fabbb",
         "comment": "{comments.color.base.value.comment}",
         "legacy": true
       },
-      "orca": {
-        "value": "222222",
-        "comment": "{comments.color.base.value.comment}",
-        "legacy": true
-      },
-      "sunset": {
+      "unionblue": {
```

Comment:
Needs a new category name

Reply:
One of the blue/gray colors that don't really have a home. Current use:

* https://github.com/AlaskaAirlines/OrionDesignTokens/blob/hiddenColorRefactor/src/color/icon.json#L6
* https://github.com/AlaskaAirlines/OrionDesignTokens/blob/hiddenColorRefactor/src/color/text.json#L22